conformance/features/httpgateway/httpgateway.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package httpgateway
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/cucumber/godog"
+	"github.com/cucumber/messages-go/v10"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	tstate "sigs.k8s.io/gateway-api/conformance/state"
+	"sigs.k8s.io/gateway-api/conformance/utils"
+)
+
+var (
+	state *tstate.Scenario
+)
+
+// InitializeScenario configures the Feature to test
+func InitializeScenario(ctx *godog.ScenarioContext) {
```

Comment:
```
?       sigs.k8s.io/gateway-api/conformance/features/httpgateway        [no test files]
=== RUN   TestSuite
I0607 15:08:10.180706  827919 conformance_test.go:109] testFeature ===> ../scenarios/httpgateway/httpgateway.feature

No scenarios
No steps
6.333µs
--- PASS: TestSuite (0.00s)
```

Not sure what I am doing wrong, but this seems skipped?

Reply:
Not doing anything wrong, the latest version I'd pushed didn't have that linked together yet. Should work well now.